File: array/remove-duplicates-from-sorted-array.kt
// Remove Duplicates from Sorted Array

class Solution {
    fun removeDuplicates(nums: IntArray): Int {
        var insertingIndex = 1
        for (i in 1 until nums.size) {
            if (nums[i - 1] != nums[i]) {
                nums[insertingIndex] = nums[i]
                insertingIndex++
            }
        }
        return insertingIndex
    }
}

File: array/remove-duplicates-from-sorted-array.ts
// Remove Duplicates from Sorted Array

function removeDuplicates(nums: number[]): number {
    let insertingIndex = 1;
    for (let i = 1; i < nums.length; i++) {
        if (nums[insertingIndex - 1] != nums[i]) {
            nums[insertingIndex] = nums[i];
            insertingIndex++;
        }
    }
    return insertingIndex;
};
